src/xenia/ui/window_gtk.cc

```diff
@@ -350,18 +404,18 @@ bool GTKWindow::HandleKeyboard(GdkEventKey* event) {
   bool ctrl_pressed = modifiers & GDK_CONTROL_MASK;
   bool alt_pressed = modifiers & GDK_META_MASK;
   bool super_pressed = modifiers & GDK_SUPER_MASK;
+  uint32_t key_char = gdk_keyval_to_unicode(event->keyval);
   auto e =
       KeyEvent(this, event->hardware_keycode, 1, event->type == GDK_KEY_RELEASE,
                shift_pressed, ctrl_pressed, alt_pressed, super_pressed);
   switch (event->type) {
     case GDK_KEY_PRESS:
       OnKeyDown(&e);
+      if (key_char > 0) OnKeyChar(&e);
```

Comment:
Xenia code style uses `{}` for all conditionals.

Reply:
Done in two places